tests/base.py

```diff
@@ -106,3 +107,11 @@ def tearDown(self) -> None:
         self._conn.close()
         self._process.terminate()
         return super().tearDown()
+
+    @property
+    def gdb_version(self) -> Tuple[int, int]:
+        res = tuple(
+            map(int, re.search(r"(\d+)[^\d]+(\d+)", self._gdb.VERSION).groups())
+        )
+        assert len(res) >= 2
```

Comment:
The `groups` call will fail if there aren't exactly two matches. `re.search` will return `None` if the version string doesn't match against the regex, and then `groups()` will raise an `AttributeError`.

You could instead assert that `.search` returns a match, and then check the length of `groups` (it will always be 2), or just drop this stuff.

Reply:
I think we should drop this. The `search.groups()` will already raise the exception. This assert is impossible to fail.